packages/api/src/functions/dbAuth/DbAuthHandler.ts

```diff
@@ -149,20 +160,20 @@ type Params = {
   username?: string
   password?: string
   method: AuthMethodNames
-  [key: string]: unknown
+  [key: string]: any
```

Comment:
The answer is: Because further down, in `_createUser` we destructure `this.params` and store most of it in `...userAttributes`. We then pass `userAttributes` to `signup.handler`, and that function doesn't handle `unknown`.

So now I have a proposal to get rid of this now
```suggestion
```

And instead use `UserAttributes`.
```
type Params = UserAttributes & {
  username?: string
  password?: string
  method: AuthMethodNames
}
```
(notice how it makes a union of `UserAttributes` and the other attributes listed).

This is what I'm most unsure about, because now we've said that all extra param keys can only have string values.  This should be fine, because we already assign the extra keys to `Record<string, string>` in the `_createUser` function. But please let me know if you don't agree

Reply:
I think you were meant to push up some changes here? Mind resolving this conversation? I don't quite follow what you were saying here...

Reply:
I'll do a followup PR instead. Makes it easier to reason about my proposed changes when we can see them in isolation